src/pages/workspaces/workspace/Data.js

```diff
@@ -393,14 +394,13 @@ class EntitiesContent extends Component {
         )(async () => {
           const str = this.buildTSV(columnSettings, entities)
           await clipboard.writeText(str)
-          this.setState({ copied: true })
+          notify('success', 'Successfully copied to clipboard', { timeout: 3000 })
```

Comment:
I believe that since we're using `withErrorReporting`, that this `notify('success')` will show up regardless of whether it was successful in copying the data. Is there a way to block this `notify` if the `await` is not successful?

Reply:
Or maybe it makes sense to include an optional prop to `withErrorReporting` which is a `success` message? I've come across this conundrum before.

Reply:
I believe since `withErrorReporting` is a try-catch, if the promise fails, it'll go straight to the error handler and what comes after it won't get hit (the notify).